ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,366 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import rdflib
+import urlparse
+import sets
+import os
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+
+
+class G2PDataset:
+    """
+    An rdf object store.  The cancer genome database
+    [Clinical Genomics Knowledge Base]
+    (http://nif-crawler.neuinfo.org/monarch/ttl/cgd.ttl)
+    published by the Monarch project was the source of Evidence.
+    """
+
+    def __init__(self, setName, relativePath, dataDir):
+        """
+        Initialize dataset, using the passed dict of sources
+        [{source,format}] see rdflib.parse() for more
+        """
+        self._sources = [os.path.join(relativePath, f)
+                         for f in os.listdir(relativePath)]
+        self._searchQuery = """
+            PREFIX OBAN: <http://purl.org/oban/>
+            PREFIX OBO: <http://purl.obolibrary.org/obo/>
+            PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+            PREFIX faldo: <http://biohackathon.org/resource/faldo#>
+            PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+            SELECT %PROPERTIES%
+                WHERE {
+                    ?s    a OBAN:association .
+                    ?s  OBAN:association_has_subject ?l .
+                    ?l  rdfs:label ?location_label  .
+                    %LOCATION%
+                    ?s  OBO:RO_has_environment  ?drug .
+                    ?drug  rdfs:label ?drug_label  .
+                    ?s  OBAN:association_has_object  ?d .
+                    ?d  rdfs:label ?disease_label  .
+                    ?d rdf:type ?disease .
+                    ?s  OBAN:association_has_object_property  ?evidence .
+                    OPTIONAL {  ?evidence  rdfs:label ?evidence_label } .
+              %FILTER%
+            }
+            """
+
+        # initialize graph
+        self._rdfGraph = rdflib.ConjunctiveGraph()
+
+        # load with data
+        for source in self._sources:
+            if source.endswith('.ttl'):
+                self._rdfGraph.parse(source,
```

Comment:
I was just wondering if we could put in 
```python
                   self._rdfGraph.parse(source, format='n3')
```
The line doesn't look that wide to me.